lib/index.d.ts

```diff
@@ -290,6 +290,21 @@ declare namespace Realm.Sync {
         isAdmin: boolean;
     }
 
+    class Credentials {
```

Comment:
In Java this is called `SyncCredentials` and `SyncUser`, here it is called `Credentials` and `User` but I guess that is due it being inside the Realm.Sync namespace.